### PR TITLE
More NA opcodes

### PR DESCRIPTION
**Pull request by TeraProxy**
Not all tested yet, that's why I'll only create a pull request for now.

### CODE REVIEW

File: map/protocol.323308.map [hunk 1]
```diff
@@ -180,18 +212,24 @@ S_SYSTEM_MESSAGE_LOOT_ITEM = 22100
 S_SYSTEM_MESSAGE_LOOT_SPECIAL_ITEM = 47474
 S_TARGET_INFO = 31707
 S_TELEPORT_TO_CAMP = 24870
+S_TRADE_BROKER_DEAL_INFO_UPDATE = 21127
 S_TRADE_BROKER_DEAL_SUGGESTED = 54056
+S_TRADE_BROKER_REQUEST_DEAL_RESULT = 49381
```
Comment (Mkalo) [inline map/protocol.323308.map]: I got this, might be wrong
S_TRADE_BROKER_REQUEST_DEAL_RESULT = 22296

Comment (TeraProxy) [reply map/protocol.323308.map]: Yeah, looks like you are correct

**Comment by pinkipi** [comment]
S_SOCIAL_CANCEL and S_TRADE_BROKER_SUGGEST_DEAL are wrong, the rest are correct.

**Comment by TeraProxy** [comment]
Thanks for the verification!

**Pull request merged**